polkadot-protocol/architecture/parachains/consensus.md

```diff
@@ -0,0 +1,68 @@
+
+At the core of parachain functionality are collators, specialized nodes that sequence transactions into blocks and maintain the parachain’s state. Collators optimize Polkadot’s architecture by offloading state management from the relay chain, allowing relay chain validators to focus solely on validating parachain blocks.
+
+This guide explores how parachain consensus works, including the roles of collators and validators, and the steps involved in securing parachain blocks within Polkadot’s scalable and decentralized framework.
+
+## The Role of Collators
+
+Collators are responsible for sequencing end-user transactions into blocks and maintaining the current state of their respective parachains. Their role is akin to Ethereum’s sequencers but optimized for Polkadot's architecture.
+
+Key responsibilities include:
+
+- **Transaction sequencing** - organizing transactions into [Proof of Validity (PoV)](https://wiki.polkadot.network/docs/glossary#proof-of-validity){target=\_blank} blocks
+- **State management** - maintaining parachain states without burdening the relay chain validators
+- **Consensus participation** - sending PoV blocks to relay chain validators for approval
+
+## Consensus and Validation
+
+Parachain consensus operates in tandem with the relay chain, leveraging Nominated Proof of Stake (NPoS) for shared security. The process ensures parachain transactions achieve finality through the following steps:
+
+- **Packaging transactions** - collators bundle transactions into PoV blocks (parablocks)
+- **Submission to validator** - parablocks are submitted to a randomly selected subset of relay chain validators, known as paravalidators
+- **Validation of PoV Blocks** - paravalidators use the parachain’s state transition function (already available on the relay chain) to verify transaction validity
+- **Backing and inclusion** - if a sufficient number of positive validations are received, the parablock is backed and included via a para-header on the relay chain
+
+The following sections describe the actions taking place during each stage of the process. 
+
+### Path of a Parachain Block
+
+Polkadot achieves scalability through execution sharding, where each parachain operates as an independent shard with its own blockchain and state. Shared security for all parachains is provided by the relay chain, powered by [Nominated Proof of Staking (NPoS)](/polkadot-protocol/glossary/#nominated-proof-of-stake-npos){target=\_blank}. This framework allows parachains to focus on transaction processing and state management, while the relay chain ensures validation and finality.
+
+The journey parachain transactions follow to reach consensus and finality can be described as follows:
```

Comment:
I don't love these sentences that lead to section headers. But it's fine for now. I also think the following section could benefit from an ordered list instead of an unordered list as it seems to be in sequential order. Lastly, a diagram might be nice here, but these are all things I am not going to be picky about at this time haha